packages/dnb-design-system-portal/src/docs/uilib/components/info-card/Examples.tsx

```diff
@@ -104,6 +104,16 @@ export const InfoCardCentered = () => (
   </ComponentBox>
 )
 
+export const InfoCardWithoutDropShadow = () => (
+  <ComponentBox data-visual-test="info-card-drop-shadow">
```

Comment:
```suggestion
  <ComponentBox>
```
If we need a visual test on this, we need to add it on the test file as well.
But I think, we probably don't need a visual test for this case as its a minor visual detail.